include/retdec/fileformat/utils/tlsh/tlsh_impl.h

```diff
@@ -0,0 +1,164 @@
+/*
```

Comment:
For @PeterMatula to consider: Do we really want a verbatim copy of [tlsh](https://github.com/trendmicro/tlsh) in our repo? Other dependencies are pulled through external projects. Now, it looks as if tlsh was part of RetDec (AFAIK, we use `src` and `include` for our own code and `deps` for dependencies). If you have agreed with @HoundThe on that, then OK, but I just wanted to be sure.

Reply:
Since adding more external deps is PITA, and tlsh is only a few C++ sources, we agreed on inlining them to our repo. But definitely not at their current location. They should go into `/deps/` as other external projects. I will for sure mention this in my review.